apps/et/et-sya/demo.yaml

```diff
@@ -8,4 +8,3 @@ spec:
   values:
     nodejs:
       ingressHost: et-sya.demo.platform.hmcts.net
-      ingressClass: traefik-private
```

Comment:
this is valid on demo, see https://hmcts.github.io/ways-of-working/environments/external-access.html#external-access-in-demo

Reply:
Corrected